roomserver/internal/query/query.go

```diff
@@ -138,18 +138,39 @@ func (r *Queryer) QueryEventsByID(
 	request *api.QueryEventsByIDRequest,
 	response *api.QueryEventsByIDResponse,
```

Comment:
This API needs to make it very clear that you cannot pull events for multiple rooms in a single request.